tests/src/helpers.c

```diff
@@ -295,25 +295,38 @@ void mbedtls_test_err_add_check( int high, int low,
      *                                                shhhhhhhhlllllll
      *
      * s = sign bit.
-     * h = high level error code (includes high and module error codes).
+     * h = high level error code (includes high level module ID (bits 12..14)
+     *     and module-dependent error code (bits 7..11)).
```

Comment:
Bits 7..11 and 12..14 aren't really separate anymore: we ran out of “module IDs” a while ago, and SSL overflowed. Not a blocker since you're just reflecting the somewhat obsolete description in `error.h`.

Reply:
I agree that they aren't really separate however as we have not updated the documentation in `error.h` I consider this out of scope for this PR.